extensions/tmtv/src/commandsModule.js

```diff
@@ -553,7 +553,10 @@ const commandsModule = ({
         commandsManager.runCommand('setViewportColormap', {
           viewportIndex,
           displaySetInstanceUID: ptDisplaySet.displaySetInstanceUID,
-          colormap,
+          colormap: {
+            name: colormap,
+            opacityMapping: [{ value: 0.1, opacity: 0.9 }],
```

Comment:
Why this particular opacityMapping?

Reply:
No idea - that is what is provided elsewhere in the code, without comment there either.